src/e2e-test/features/admin.feature

```diff
@@ -30,6 +31,7 @@ Feature: Admin - Validate profile setting
 
   @ADMIN_TEST
   Scenario: Allow user to save valid preference at instance level after fixing error
+    Given Login if required
```

Comment:
Is there an equivalent of `before` or `beforeEach` in Cucumber?

Reply:
I think there is but I am not sure if it's for all feature files or can be tied to a single feature file (we don't call the `loginIfRequired` method in all feature files). I'll update once I have a better idea